news/2022-09-28-new-featured-artist-feint.md

```diff
@@ -0,0 +1,233 @@
+</audio>
+
+### Feint - Sky Dance
+
+Experience [this osu! map](https://osu.ppy.sh/beatmapsets/146251#osu/414881) by [Secretpipe](https://osu.ppy.sh/users/2208964) or [this osu!taiko map](https://osu.ppy.sh/beatmapsets/234125#taiko/588823) by [Aldwych](https://osu.ppy.sh/users/1416484)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Songs/Feint%20-%20Sky%20Dance.mp3" type="audio/mpeg">
+</audio>
+
+### Doxx & Feint - Mind In Motion
+
+Try out [this osu! map](https://osu.ppy.sh/beatmapsets/222565#osu/538573) by [Secretpipe](https://osu.ppy.sh/users/2208964)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Doxx%20%26%20Feint%20EP%2FDoxx%20%26%20Feint%20-%20Mind%20In%20Motion.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - One Last Time
+
+Check out [this osu! map](https://osu.ppy.sh/beatmapsets/219941#osu/522818) by [Secretpipe](https://osu.ppy.sh/users/2208964)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Songs/Feint%20-%20One%20Last%20Time.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - Horizons (feat. Veela)
+
+Relax with [this osu! map](https://osu.ppy.sh/beatmapsets/288528#osu/651244) by [Milan-](https://osu.ppy.sh/users/1052994)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Liquicity%20Memoirs%2FFeint%20-%20Horizons.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - Homebound
+
+Try playing [this classic osu! map](https://osu.ppy.sh/beatmapsets/276967#osu/627687) by [diodex](https://osu.ppy.sh/users/3513559) or [this osu!taiko map](https://osu.ppy.sh/beatmapsets/1307471#taiko/2710693) by [HEAVENLY MOON](https://osu.ppy.sh/users/13681283)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Homebound/Feint%20-%20Homebound.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - Phosphor (feat. Miyoki)
+
+Tap to something other than 174 BPM in [this osu! map](https://osu.ppy.sh/beatmapsets/255737#osu/622571) by [Secretpipe](https://osu.ppy.sh/users/2208964)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Phosphor/Feint%20-%20Phosphor%20(feat.%20Miyoki).mp3" type="audio/mpeg">
+</audio>
+
+### Feint - 161914
+
+Play [this short osu! map](https://osu.ppy.sh/beatmapsets/224157#osu/523365) by [Aka](https://osu.ppy.sh/users/1307553)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Songs/Feint%20-%20161914.mp3" type="audio/mpeg">
+</audio>
+
+### Doxx & Feint - The Pink Fields
+
+Attempt some classic jumps with [this osu! map](https://osu.ppy.sh/beatmapsets/329290#osu/729986) by [Bearizm](https://osu.ppy.sh/users/3495259)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Doxx%20%26%20Feint%20EP%2FDoxx%20%26%20Feint%20-%20The%20Pink%20Fields.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - Vagrant (feat. Veela)
+
+Alternatively, attempt some modern jumps with any of these beatmaps:
```

Comment:
last time i checked, there were no jumps in taiko

Reply:
Eh, I think it's okay.